.env

```diff
@@ -0,0 +1,3 @@
+NETWORK="xdai"
+PUBLIC_ADDRESS="0x90F8bf6A479f320ead074411a4B0e7944Ea8c9C1"
+PRIVATE_KEY="0x4f3edf983ac636a65a842ce7c78d9aa706d3b113bce9c46f30d7d21715b23b1d"
```

Comment:
These are the default keys that everyone knows when running `ganache-cli -d` (deterministic ganache).

I still think there should be some clean way to recover the public address from the private key without having to include both in the env file (but we can save this for a another time).